.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -0,0 +1,194 @@
+name: Arm64-CI
```

Comment:
May be nitpicking, but please remove the dash (`Arm64 CI`), so the workflow naming is consistent in actions tab.